libs/blocks/modal/modal.js

```diff
@@ -34,9 +34,7 @@ export function sendAnalytics(event) {
 function closeModal(modal) {
   const { id } = modal;
   const closeEvent = new Event('milo:modal:closed');
-  if (document.body.classList.contains('modal-open')) {
-    document.body.classList.remove('modal-open');
-  }
+  document.body.classList.remove('commerce-modal-open');
```

Comment:
Should the general modal close the commerce modal?
I would imagine you listen to the event and close the modal based on that

Reply:
Come to think of it, @mokimo does have a point. Why is this logic here instead of the commerce code? There's already a `milo:modal:closed` event that could be listened to on the commerce side. When it fires, you can remove the commerce-specific class. Same for adding the class, you should be able to dispatch an event from the general modal logic here and tie into it in the commerce logic.